lib/schemaUtils.js

```diff
@@ -2278,7 +2357,12 @@ module.exports = {
     switch (options.requestNameSource) {
       case 'fallback' : {
         // operationId is usually camelcase or snake case
-        reqName = operation.summary || utils.insertSpacesInName(operation.operationId) || reqUrl;
+        if (fromWebhooks) {
+          reqName = utils.insertSpacesInName(operation.operationId) || reqUrl;
```

Comment:
we need to add the fallback value right?

Reply:
here i've added a last fallback value because neither the operationId and summary will be there in all cases